### PR TITLE
Control: documentation updates for Apollo 5.0 new features

### PR DESCRIPTION
**Pull request by kevin-y-wang**

### CODE REVIEW

File: modules/control/README.md [hunk 1]
```diff
@@ -3,6 +3,13 @@
 ## Introduction
 Based on the planning trajectory and the car's current status, the Control module uses different control algorithms to generate a comfortable driving experience. The Control module can work both in normal and navigation modes.
 
+Apollo 5.0 Control has following new features:
+ * **Unlock Low Speed / Reverse Driving Scenarios**: By introducing a series of control methodologies ("look-ahead" controller, etc.) and upgrading the control-oriented dynamic models, the control module significantly improves the control performance under low speed and reverse driving modes, so as to support the valet parking, pull-over and many other practical driving scenarios.
+ * **Reference/Feedback Signal Coordinate Transformation**: The coordinate of the planning trajectory ("reference" signal of control system) and the vehicle states ("feedback" signal of the control system) can be transformed between the center of the rear-axis and the center of mass of the vehicle, to develop the control performance with respect to various driving scenarios.
```
Comment (natashadsouza) [inline modules/control/README.md]: Keep the / consistent as the previous point

File: modules/control/README.md [hunk 2]
```diff
@@ -3,6 +3,13 @@
...
+ * **Optimized MPC Controller Algorithm**: The structure design of the MPC (Model Predictive Control) algorithm is optimized and consequently, the control accuracy with MPC controller, particularly at sharp turning scenarios, gets improved to a large extent.
```
Comment (natashadsouza) [inline modules/control/README.md]: The structural design of the MPC (Model Predictive Control) algorithm is optimized to greatly improve the control accuracy with the MPC controller, especially in scenarios that involve sharp turns.

File: modules/control/README.md [hunk 3]
```diff
@@ -3,6 +3,13 @@
...
+ * **Unlock Low Speed / Reverse Driving Scenarios**: By introducing a series of control methodologies ("look-ahead" controller, etc.) and upgrading the control-oriented dynamic models, the control module significantly improves the control performance under low speed and reverse driving modes, so as to support the valet parking, pull-over and many other practical driving scenarios.
+ * **Reference/Feedback Signal Coordinate Transformation**: The coordinate of the planning trajectory ("reference" signal of control system) and the vehicle states ("feedback" signal of the control system) can be transformed between the center of the rear-axis and the center of mass of the vehicle, to develop the control performance with respect to various driving scenarios.
+ * **Optimized MPC Controller Algorithm**: The structure design of the MPC (Model Predictive Control) algorithm is optimized and consequently, the control accuracy with MPC controller, particularly at sharp turning scenarios, gets improved to a large extent.   
+ * **RTK Player Update**: The flexible switching of the gear locations (among forward drive, reverse drive, etc.) is enabled under the RTK (Real-Time-kinematics) test mode
```
Comment (natashadsouza) [inline modules/control/README.md]: The flexible switching of the vehicle gear (forward drive, reverse drive, etc.) is enabled under the RTK (Real-Time-Kinematics) test mode

**Comment by Capri2014** [comment]
@natashadsouza Please help review again

File: modules/control/README.md [hunk 4]
```diff
@@ -3,6 +3,11 @@
 ## Introduction
 Based on the planning trajectory and the car's current status, the Control module uses different control algorithms to generate a comfortable driving experience. The Control module can work both in normal and navigation modes.
 
+Apollo 5.0 Control has following new features:
+ * **Low Speed / Reverse Driving**: The control module now enables more accurate low-speed forward and backward driving, and therefore supports the valet parking, pull-over and other practical driving scenarios.
```
Comment (natashadsouza) [inline modules/control/README.md]: The control module ensures accurate low-speed forward and backward driving, which supports complex and practical driving scenarios like valet parking and pull-over to name a few.

Comment (kevin-y-wang) [reply modules/control/README.md]: Fixed per comment. Thank you.

**Pull request merged**